### PR TITLE
Switch CI_release.yml to 2024.2.pre0

### PR DESCRIPTION
**Pull request by PProfizi**
Test release pipeline: https://github.com/ansys/pydpf-core/actions/runs/7900013659

Latest: https://github.com/ansys/pydpf-core/actions/runs/7914266994

### CODE REVIEW

**Review by BClappe — APPROVED** [review]

File: .github/workflows/ci_release.yml [hunk 1]
```diff
@@ -83,38 +86,47 @@ jobs:
...
+      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.pre0' }}
+    secrets: inherit
+
+  retro_241:
+    name: "retro 241"
+    uses: ./.github/workflows/tests.yml
+    with:
+      ANSYS_VERSION: "241"
+      python_versions: '["3.9"]'
+      DOCSTRING: false
```
Comment (rafacanton) [inline .github/workflows/ci_release.yml:129]: @PProfizi retro 241 should point to sp01, right? (unless 241 now is equal to sp01)

Comment (PProfizi) [reply .github/workflows/ci_release.yml:129]: @rafacanton good catch!

File: .github/workflows/ci_release.yml [hunk 2]
```diff
@@ -83,38 +86,47 @@ jobs:
...
-      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.sp01' }}
+      standalone_suffix: ${{ github.event.inputs.standalone_branch_suffix || '.pre0' }}
+    secrets: inherit
+
+  retro_241:
+    name: "retro 241"
+    uses: ./.github/workflows/tests.yml
+    with:
+      ANSYS_VERSION: "241"
+      python_versions: '["3.9"]'
```
Comment (PProfizi) [inline .github/workflows/ci_release.yml:128]:
```suggestion
      python_versions: '["3.9"]'
      standalone_suffix: '.sp01'
```

**Review by rafacanton — APPROVED** [review]

**Comment by PProfizi** [comment]
Trying to skip a problematic doctest line for Ubuntu on 3.9 with 2024.2.pre0: https://github.com/ansys/pydpf-core/actions/runs/7904444681/job/21577966159

**Pull request merged**